1_fetch/src/create_grids.R

```diff
@@ -0,0 +1,62 @@
+create_conus_grid <- function(cellsize, year = 2020, progress_bar = FALSE){
+  #' 
+  #' @description Function to create a big grid of boxes to use for chunked
+  #' data queries
+  #' 
+  #' @param cellsize the target cell size of each box in map units (here, degrees);
```

Comment:
might want to explain that it needs two values - size in x direction and size in y direction

Reply:
Good point, I've made that change in [a3046a2](https://github.com/USGS-R/ds-pipelines-targets-example-wqp/pull/15/commits/a3046a240e3132472ed3c187f70d0128d642b375).